src/test/java/uk/co/probablyfine/matchers/StreamMatchersTest.java

```diff
@@ -17,307 +17,307 @@
 import static org.hamcrest.Matchers.lessThanOrEqualTo;
 import static org.hamcrest.Matchers.not;
 
-public class StreamMatchersTest {
-    @Test
-    public void equalTo_failureDifferingSingleItem() throws Exception {
+class StreamMatchersTest {
+
+    void equalTo_failureDifferingSingleItem() {
```

Comment:
Is there a missing `@Test` that got deleted by mistake on this test?

Reply:
Yes! Good catch. Will fix!